src/views/Form/Form.scss

```diff
@@ -22,6 +22,10 @@
 
     li {
       margin: 0 4rem;
+
+      p {
+        padding: 0;
+      }
     }
 
     @media screen and (max-width: 1380px) {
```

Comment:
It is definitely [valid syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Syntax). Will have to research again why the Github syntax highlighter doesn't like some of these styles.